.github/workflows/buildx_and_publish.yml

```diff
@@ -0,0 +1,78 @@
+name: Docker multiarch publish
+
+on: push
+
+env:
+  REPOSITORY: moodle-php-apache
+  DOCKERHUB_OWNER: moodlehq
+  GH_OWNER: moodlehq
+
+jobs:
+  Build:
+    # Completely avoid forks to try this workflow.
+    if: github.repository_owner == 'moodlehq'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Calculate the tags to be pussed to the registries.
+      - name: Calculate image tag names
+        id: calculatetags
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ${{ env.DOCKERHUB_OWNER }}/${{ env.REPOSITORY }}
+            ghcr.io/${{ env.GH_OWNER }}/${{ env.REPOSITORY }}
+          flavor: |
+            latest=false
+          tags: |
+            type=raw,value={{branch}}
+            type=match,pattern=(\d+.\d+),value={{branch}}
+
+      # https://github.com/docker/setup-qemu-action#usage
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      # https://github.com/marketplace/actions/docker-setup-buildx
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      # https://github.com/docker/login-action#docker-hub
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # https://github.com/docker/login-action#github-container-registry
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GH_USERNAME }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # https://github.com/docker/build-push-action#multi-platform-image
+      - name: Build and push to Docker Hub and Github registries
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ steps.calculatetags.outputs.tags }}
+
+      # https://github.com/peter-evans/dockerhub-description
+      # It's horrible that we need to use password here instead of PAT, because
+      # that's only available via CLI what defeats 2FA. Anyway, we need to
+      # auto-update de description, so using it (till available via PAT).
```

Comment:
Small typo here, "de" => "the".

HTH,
Matteo

Reply:
Thanks, Matteo, Spanglish!

I've amended the 7.3 and 7.4 branches and added typo-fix commit to the, already rolled, 8.0 and mater branches.

Ciao :-)

Reply:
TNX to you!

> Spanglish!

I know that feeling being myself not native English too... 😉.

HTH,
Matteo